### DISCUSSION TITLE
web控制页面进不去了怎么办

### DISCUSSION THREAD

**Discussion by wolf7689** [istoreos/istoreos#927] [Q&A]
在上个月升级完应该是0714版本后web控制页就进不去了。系统是装在PVE里面作为旁路由还在正常运行的，包括里面的docker也在运行，但是不能修改十分不方便。请问这种情况我要怎么修复。

**Comment by wolf7689** [comment]
我搞定了，uhttpd -p 80 -h /www

**Reply by jjm2473** [reply]
是不是之前使用了nginx，将nginx卸载重装就行了

**Reply by jjm2473** [reply]
issue那边有详细的命令，自己搜一下
